serverless/serverless
# Add rollback function support

## What did you implement:

Closes #3488

Add support for rolling back a single function.

Standing on the shoulders of giants here --> Shoutouts to @marcy-terui who did the heavy lifting in ( #2672) 🙌 

## How did you implement it:

Add a new `rollback function` command with its corresponding plugin.

## How can we verify it:

1. Deploy a service
2. Run `serverless deploy list functions` to see all the functions and their versions
3. Run `serverless rollback function -f <name-of-the-function> -v <version>` to rollback a function to the requestd version

## Todos:

- [x] Re-run shrinkwrap (because `node-fetch` was bumped)

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
